en_US/extensions/plugins.md

```diff
@@ -162,70 +175,64 @@ make rel
 ===> [emqx_plugrel] creating _build/default/emqx_plugrel/my_emqx_plugin-1.0.0.tar.gz
 ```
 
-This created a new EMQX plugin tarball  `my_emqx_plugin-1.0.0.tar.gz` that you can now upload and install to your running EMQX cluster.
+This generates a new tarball, `my_emqx_plugin-1.0.0.tar.gz`, which you can upload and deploy to your EMQX cluster.
 
-#### Write Config Schema for the plugin (Optional)
+#### Write Configuration Schema for the Plugin (Optional)
 
 {% emqxce %}
-:::tip
-EMQX Enterprise Edition features. EMQX Enterprise Edition provides comprehensive coverage of key business scenarios, rich data integration, product-level reliability, and 24/7 global technical support. Experience the benefits of this [enterprise-ready MQTT messaging platform](https://www.emqx.com/en/try?product=enterprise) today.
-:::
-{% endemqxce %}
+::: tip
 
-{% emqxce %}
-::: tip Tip
 EMQX Enterprise Edition features. EMQX Enterprise Edition provides comprehensive coverage of key business scenarios, rich data integration, product-level reliability, and 24/7 global technical support. Experience the benefits of this [enterprise-ready MQTT messaging platform](https://www.emqx.com/en/try?product=enterprise) today.
+
 :::
 {% endemqxce %}
 
-We introduced Avro Schema for plugins in EMQX Enterprise Edition 5.7.0 to provide the ability to update plugin configurations at runtime.
-Furthermore, you can declare UI in this schema, allowing the EMQX Dashboard to render configuration forms for ease of use.
+EMQX Enterprise version 5.7.0 introduced Avro Schema for plugins, enhancing the ability to update plugin configurations dynamically during runtime. This feature also allows for the declaration of a user interface in the schema, enabling the EMQX Dashboard to dynamically generate configuration forms for easier management.
```

Comment:
Avro schema is available for opensource edition, the UI for configuration management is provided in Enterprise only.

```suggestion
EMQX version 5.7.0 introduced REST API for plugin configuration management and Avro Schema for configuration validation, enhancing the ability to update plugin configurations dynamically during runtime.
For Enterprise edition, it also allows the declaration of a user interface in the schema, enabling the EMQX Dashboard to dynamically generate configuration forms for easier management.
```